src/components/Input/Input.js

```diff
@@ -95,15 +102,25 @@ Input.propTypes = {
   /**
    * Autocomplete attribute to be passed down to the <input> element.
    */
-  autoComplete: PropTypes.string
+  autoComplete: PropTypes.string,
+  /**
+   * Displays input inline.
+   */
+  inline: PropTypes.bool,
+  /**
+   * Adds bottom margin to the input.
+   */
+  margin: PropTypes.bool
 };
 
 Input.defaultProps = {
   theme: standard,
   invalid: false,
   optional: false,
   disabled: false,
-  autoComplete: 'none'
+  autoComplete: 'none',
```

Comment:
Is this a good default?

Reply:
I didn't add this :) 

@felixjung What was your reasoning when you added this?